application/src/main/java/org/togetherjava/tjbot/commands/system/BotCore.java

```diff
@@ -241,6 +242,18 @@ public void onSlashCommandInteraction(SlashCommandInteractionEvent event) {
                     .onSlashCommand(event));
     }
 
+    @Override
+    public void onCommandAutoCompleteInteraction(
+            @NotNull final CommandAutoCompleteInteractionEvent event) {
+        String name = event.getName();
+
+        logger.debug("Received auto completion from command-subcommand '{}-{}' (#{}) on guild '{}'",
+                name, event.getSubcommandName(), event.getId(), event.getGuild());
```

Comment:
what will happen if the command doesn't have a subcommand??

Reply:
Good point, I should replace this with getPath or whatever it's called (shows command, groups and subcommand)

If anyone can do this it'd be highly appreciated, not home till tomorrow evening.